tutorials/tour/basics.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Trying Scala in the Browser
+
+You can run Scala in your browser by using [ScalaFiddle](https://scalafiddle.io).
+
+1. Go to https://scalafiddle.io.
+2. Copy and paste `println("Hello, world!")` to the left pane.
+3. Hit "Run" button and see it prints "Hello, world!" on the right pane.
+
+It is a perfect way for anybody to experiment with a piece of Scala code anywhere, anytime!
+
+## Values and Variables
+
+You can define values (a.k.a. immutable variables) with the `val` keyword.
```

Comment:
Continued from https://github.com/scala/scala.github.com/pull/671#discussion_r99347365

It's tough:) So,

- we don't want to use the term "immutable variable" because it's technically incorrect.
- I personally think "computed expression" can be intimidating although it is correct.

Maybe we should have "Expressions" section before "Values and Variables", so that terms like "computed expression" or "result of the expression" won't be as intimidating. There, we can also state things like "almost everything in Scala is an expression" or "multiple expressions must be grouped by `{}`", and we will be able to use the term "expression" without intimidating readers for the rest of the tutorial. In the section, we can use `println` as a tool to show the result of each expression because we are not using REPL.

How do you guys think?